Source/WebKit/UIProcess/API/gtk/WebKitWebViewBase.cpp

```diff
@@ -1299,8 +1301,10 @@ static gboolean webkitWebViewBaseScrollEvent(GtkWidget* widget, GdkEventScroll*
     WebKitWebViewBase* webViewBase = WEBKIT_WEB_VIEW_BASE(widget);
     WebKitWebViewBasePrivate* priv = webViewBase->priv;
 
-    if (std::exchange(priv->shouldForwardNextWheelEvent, false))
+    if (!priv->shouldForwardNextWheelEvent.isEmpty()) {
+        priv->shouldForwardNextWheelEvent.remove(0);
```

Comment:
Elsewhere you unref the events before removing them from this vector, but here you don't. Need to be consistent. Does the vector hold ownership or not?

Reply:
Removed ref/unref code